pymeasure/instruments/keithley/keithley2000.py

```diff
@@ -58,16 +58,25 @@ class Keithley2000(Instrument, KeithleyBuffer):
     mode = Instrument.control(
         ":CONF?", ":CONF:%s",
         """ A string property that controls the configuration mode for measurements,
-        which can take the values: :code:'current' (DC), :code:'current ac', 
-        :code:'voltage' (DC),  :code:'voltage ac', :code:'resistance' (2-wire), 
-        :code:'resistance 4W' (4-wire), :code:'period', :code:'frequency', 
+        which can take the values: :code:'current' (DC), :code:'current ac',
+        :code:'voltage' (DC),  :code:'voltage ac', :code:'resistance' (2-wire),
+        :code:'resistance 4W' (4-wire), :code:'period', :code:'frequency',
         :code:'temperature', :code:'diode', and :code:'frequency'. """,
         validator=strict_discrete_set,
         values=MODES,
         map_values=True,
         get_process=lambda v: v.replace('"', '')
     )
 
+    beep_state = Instrument.control(
+        ":SYST:BEEP:STAT?",
+        ":SYST:BEEP:STAT %g",
+        """ A string property that enables or disables the system status beeper, which can take the values: :code:'enabled' and :code:'disabled'. """,
```

Comment:
Please keep the line length below 80-100 characters so that it is easy to read. The great thing about the `"""` multi-line strings is that you can put in linebreaks naturally (like in line 60 above).